featuretools/tests/entityset_tests/test_es.py

```diff
@@ -15,6 +15,19 @@
 from featuretools.entityset.serialize import SCHEMA_VERSION
 
 
+def test_normalize_time_index_as_additional_variable(es):
+    error_text = "Not moving signup_date as it is the base time index variable."
+    with pytest.raises(ValueError, match=error_text):
+        assert "signup_date" in es["customers"].df.columns
+        es.normalize_entity(base_entity_id='customers',
+                            new_entity_id='cancellations',
+                            index='cancel_reason',
+                            make_time_index='signup_date',
+                            additional_variables=['signup_date'],
+                            copy_variables=[])
+        assert "signup_date" in es["customers"].df.columns
```

Comment:
This line will not get hit due to the previous one raising an error

Reply:
??

Reply:
The `normalize_entity` call will raise an error, which stops code execution within the with statement.  If we wanted the final assert statement to be run after the error to confirm we didn't remove the signup date column, we would need to move the assert out of the `with pytest.raises` block